acceptance/ui/features/services.feature

```diff
@@ -29,4 +29,183 @@ Feature: Service Management
       And I should see "Public Endpoints"
       And VHost "table://services/childServiceWithVHost/vhostName" should exist
       And Public Port "table://services/childServiceWithVHost/publicPortNumber" should exist
-      And I should see "table://services/childServiceWithVHost/configFile"
\ No newline at end of file
+      And I should see "table://services/childServiceWithVHost/configFile"
+
+  @services_endpoint
+  Scenario: Public Endpoint dialog - check all labels
+    When I view the details of "table://services/topService/name" in the "Applications" table
+      And I view the details of "table://services/childServiceWithVHost/name" in the "Services" table
+      And I click "Add Public Endpoint"
+    Then I should see the Add Public Endpoint dialog
+      And I should see "Type:"
+      And I should see "Port"
+      And I should see "VHost"
+      And I should see "Service - Endpoint:"
+      And I should see "Host:"
+      And I should see "Port:"
+      And I should see "Protocol:"
+      And I click "Cancel"
+
+  @services_endpoint
+  Scenario: Public Endpoint dialog - VHost mode visible fields
+    When I view the details of "table://services/topService/name" in the "Applications" table
+      And I view the details of "table://services/childServiceWithVHost/name" in the "Services" table
+      And I click "Add Public Endpoint"
+      And I should see the Add Public Endpoint dialog
+      And I choose VHost type
+    Then I should see the Add Public Endpoint dialog
```

Comment:
Not really. Not sure if we need to test visible labels. We can remove this step.